docs/src/examples/components/Menu/Variations/index.tsx

```diff
@@ -9,6 +9,11 @@ const Variations = () => (
       description="A vertical menu displays elements vertically."
       examplePath="components/Menu/Variations/MenuExampleVertical"
     />
+    <ComponentExample
+      title="Vertical &amp; Pointing"
+      description="A vertical menu displays elements vertically and point to show its relationship to nearby content."
+      examplePath="components/Menu/Variations/MenuExampleVerticalPointing"
```

Comment:
Please remove `title` and `description` there, it's a simple variation of mixed props.

Reply:
@layershifter why not keep the title at least? users will get confused

Reply:
We should show more in less examples, for example:
https://react.semantic-ui.com/collections/menu/#types-vertical

May be we can even combine, `MenuExampleVertical` and `MenuExampleVerticalPointing` to a single example as it's done there:
https://stardust-ui.github.io/react/components/menu#types-pointing

Reply:
Should mention that in #695 was done the great clean up in the same direction.

Reply:
> We should show more in less examples, for example:
https://react.semantic-ui.com/collections/menu/#types-vertical

I don't really like that TBH, I have to open the example and looks what props have changed.

> May be we can even combine, `MenuExampleVertical` and `MenuExampleVerticalPointing` to a single example as it's done there:
https://stardust-ui.github.io/react/components/menu#types-pointing

That's a bit better but then I need to write different set of steps in `docs/src/examples/components/Menu/Variations/MenuExampleVerticalPointing.shorthand.steps.ts`

to select the 2nd menu

+ I know @miroslavstastny is not a fan of having multiple examples in one screenshot:
https://github.com/stardust-ui/react/issues/992#issuecomment-468228767

What about moving the title Vertical & Pointing to description and removing the title then?

Reply:
I'm not sure that `description` provides something useful in this case.

/cc @mnajdova @kuzhelov

Reply:
### 1. now
![screenshot 2019-02-28 at 18 06 11](https://user-images.githubusercontent.com/5442794/53584427-d44bc200-3b83-11e9-85c4-56f126ac9550.png)

### 2. @layershifter proposal
![screenshot 2019-02-28 at 18 06 40](https://user-images.githubusercontent.com/5442794/53584436-d7df4900-3b83-11e9-999c-e2395642b281.png)

### 3. compromise:
![screenshot 2019-02-28 at 18 07 31](https://user-images.githubusercontent.com/5442794/53584445-db72d000-3b83-11e9-8989-e338d99475b5.png)

or a short description...

@layershifter 
@mnajdova 
@kuzhelov 
@miroslavstastny 

vote? 😕

Reply:
I see the point that both of you has made... I would go with only title here, as we previously already describe what vertical and what pointing means. This way, the user will immediately know which props are provided. What do you think?

Reply:
Also, let's keep consistency with the other examples, I know that the page for the Menu component is pretty long, so try to avoid adding more text if there is already description for something already.

Reply:
Sorry for leaving so many components, I promise it's the last one :D We don't have to show all examples, if we are adding them just so we can have screen tests...

Reply:
> We don't have to show all examples, if we are adding them just so we can have screen tests...

@mnajdova good point, how can we do that?

Reply:
My vote is for "2".
In the future will be cool to make this area more compact if the example missing a `title` and `description`. Can this satisfy all?

![image](https://user-images.githubusercontent.com/14183168/53584940-79b76380-3b8d-11e9-8103-cfd5911d9aac.png)

Reply:
@layershifter now that the title is not making anything worse (we still have the same height, I would add it) We may restructure this in future, maybe use some collapsable if we want to see all variations in vertical or something like that...
@Bugaa92 you can still add files in the examples folders, but you don't need to import them in the index.tsx files of the Variations/Types and actually show them on the docs...

Reply:
let's see what we decide here first: https://github.com/stardust-ui/react/pull/983#discussion_r261310975

Reply:
added just title for now, as Marija said, it's not taking extra space and brings no harm